### ISSUE TITLE
Simplified "try it"/ "quickstart" 

### ISSUE THREAD

**Issue by erikhopf** [trinodb/trino#7743]
Brand new users could benefit greatly from an article that's focused on getting a novice user up and running in 10 minutes or less. Ideally this guide includes:

* Easiest install path
* Most support across OS (Linux, macOS, and Windows) -- Remember, this is just to try the service, not to run this at scale.
* Give the reader an "Ah-ha" moment (or two or three) - Make me want to use the tool
* Give me the directions I need to be dangerous. Can I make the leap from try it to "something else"

I'll pull a PR together for this and we can pick it apart. I have thick skin :smile:

**Comment by hashhar** [comment]
@bitsondatadev has had this on his wishlist for quite a while now IIRC.

**Comment by bitsondatadev** [comment]
That's correct @hashhar! I have this [getting started repo](https://github.com/bitsondatadev/trino-getting-started) going, eventually to pave the way for tutorials maintained by the community and put on [the Getting Started page](https://trino.io/download.html). 

Having a page like this in the docs will definitely be helpful as we can always point to the environment setup needed for Trino, rather than various links to Docker.

Thanks for submitting this @erikhopf! If you're interested in doing more around this I'd like to get more details on your ideas  on the bullet points above.

**Comment by erikhopf** [comment]
@bitsondatadev - Sounds like a plan.
